tsdb/querier_bench_test.go

```diff
@@ -58,9 +57,11 @@ func BenchmarkQuerier(b *testing.B) {
 	}
 	require.NoError(b, app.Commit())
 
-	ir, err := h.Index()
-	require.NoError(b, err)
 	b.Run("Head", func(b *testing.B) {
+		ir, err := h.Index()
+		require.NoError(b, err)
+		defer ir.Close()
```

Comment:
Nit: You also could check for an error here, in case it might fail in the future.

Currently hardcoded to return nil:

https://github.com/prometheus/prometheus/blob/95e705612c1d557f1681bd081a841b78f93ee158/tsdb/head_read.go#L54

Reply:
Good point, @simonswine.

Reply:
Fixed.